src/intTest/resources/application.yml

```diff
@@ -118,4 +118,7 @@ nhais:
       3WJ4bVSmp3MOQ2EF4zrAl4fXEd4lS3JxKtzbHFbSogAmPFnuP8wIDWTDgfEq
       -----END RSA PRIVATE KEY-----}
     recipientCodes: ${NHAIS_MESH_RECIPIENT_CODES:XX11=A91561OT001
-      TES5=A91561OT002}
\ No newline at end of file
+      TES5=A91561OT002}
+  scheduler:
+    overlapIntervalInSeconds: ${NHAIS_SCHEDULER_OVERLAP_INTERVAL_IN_SECONDS:5}
+    intervalInMilliSeconds: ${NHAIS_SCHEDULER_INTERVAL_IN_MILLISECONDS:6000}
```

Comment:
Both are in milliseconds now

Reply:
No, they aren't :) Also `intervalInMilliSeconds` will probably be moved to mesh group soon as it's not related to scheduler, so please don't spend too much time on it

Reply:
overlap interval is in seconds, interval is in milliseconds (if I got it right, @Scheduler takes milliseconds)